docs/source/requests.md

```diff
@@ -2957,7 +2957,9 @@ A new Agreement needs to be sent instead.
 
 ### LEDGERS_FREEZE
 
-Disable specific legders. System completely ignore frozen ledgers. After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py).
+Freeze deprecated ledgers (base ledgers POOL, CONFIG, AUDIT and DOMAIN can't be frozen). System completely ignore frozen ledgers. Frozen ledgers can't be unfreeze. After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py).
```

Comment:
Deleting a frozen ledger that contains transactions has an adverse affect on being able to audit history.  Refer to https://github.com/hyperledger/indy-node/issues/1659 for additional details.

The ramifications of deleting a frozen ledger should be clearly documented.

Reply:
I agree. I made this more explicit, and removed the link to the deletion script (they can find it by following the link to the plenum documentation I included).